### DISCUSSION TITLE
Ganache - Not being able to open app

### DISCUSSION THREAD

**Discussion by danielcas1193** [smartcontractkit/full-blockchain-solidity-course-py#1270] [Q&A]
Hi everyone, 

I tried to install ganache, however when i want to open the app it gives me the following error:
![ganache error](https://user-images.githubusercontent.com/23518839/158085671-9ffccee5-87fc-41ba-abb4-e82a4455a533.PNG)

Does anyone know how to solve this?

Thanks for the help

**Comment by cromewar** [comment]
Hello @danielcas1193 
Does [this](https://support.microsoft.com/en-us/topic/kb4073393-fix-the-data-area-passed-to-a-system-call-is-too-small-error-when-you-start-a-desktop-bridge-application-on-a-sql-server-5ae0994d-023a-d32b-3aad-526500b53993) make sense for you?
